js/interface.js

```diff
@@ -2189,9 +2182,13 @@ $allowBtnFilter.click(function(event) {
 
   if (value === 'tokens') {
     var tpl = Fliplet.Widget.Templates['templates.apiTokenList'];
+    var appTokens = _.groupBy(integrationTokenList, function (token) {
+      return _.get(_.first(token.apps), 'name', 'Other');
+    });
 
     $('.tokens-list').html(tpl({
-      integrationTokenList: integrationTokenList
+      // Sort by key (app name)
+      apps: _.fromPairs(_.sortBy(_.toPairs(appTokens), 1).reverse())
```

Comment:
If we're using `_.sortBy()` but you've name one of the apps "Other", then we can't have "Other..." at the end. 

I also suggest sorting the string after they're all upper/lowercased so that A, B C don't come before a, b, c.